README.md

```diff
@@ -0,0 +1,101 @@
+<a name="readme-top">To-Do List Project</a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+    - [future features](#future-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [License](#License)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To-Do List Project] <a name="about-project"></a>
+
+[To-DoListProject] is a simple web app that allows the user to add a to-do task, check the task if it is completed and an option to delete a task and a clear all to clear all the checked tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Coded with HTML, CSS and JavaScript.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [Feature_1]: Task Creation: Users can create new tasks by providing a title or description for the task.
+- [Feature_2]: Task Organization: Tasks can be organized into categories or projects, allowing users to group related tasks together.
+
+## Future Features <a name="future-features"></a>
```

Comment:
- [ ] This section is empty please add some contents to it so your readme looks professional :+1: :sparkles: